helm/kyverno-policies-dx/templates/EnforceFallbackConfigScaledObjects.yaml

```diff
@@ -0,0 +1,31 @@
+# THIS FILE IS GENERATED WITH 'make generate' - DO NOT EDIT MANUALLY
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: enforce-fallback-config-scaledobjects
+  annotations:
+    policies.kyverno.io/title: Enforce Fallback config for ScaledObjects
+    policies.kyverno.io/category: Best Practices
+    policies.kyverno.io/severity: medium
+    policies.kyverno.io/minversion: 1.3.0
+    policies.kyverno.io/subject: ScaledObject
+    policies.kyverno.io/description: >-
+      ScaledObjects should have a fallback configuration to ensure that they can handle cases
+      where a scaler isn't working in a predictable way.
+spec:
+  validationFailureAction: Audit
+  background: true
+  rules:
+  - name: check-fallback-config
+    match:
+      resources:
+        kinds:
+        - keda.sh/v1alpha1/ScaledObject
+    validate:
+      message: >-
+        ScaledObjects must have a fallback config defined.
```

Comment:
Should we have a doc page (with a link from here) to explain why this alert, and how to fix it?

Reply:
Makes sense, I'm gonna update the keda doc page and link it here.

Reply:
Right, I see the doc update PR here: https://github.com/giantswarm/giantswarm/pull/34035
So let's wait for it to be merged, then include a link in the error message here :+1: